.github/workflows/application.yml

```diff
@@ -109,6 +109,42 @@ jobs:
                 run: vendor/bin/phpstan analyse
                 if: always() && steps.end-of-setup.outcome == 'success'
 
+    architecture-tests:
+        runs-on: ubuntu-latest
+
+        name: "Architecture tests (PHPArkitect)"
+
+        timeout-minutes: 10
+
+        strategy:
+            fail-fast: false
+            matrix:
+                php: ["7.4", "8.0"]
```

Comment:
Do we really need to test it for several PHP versions? What would you say for testing it only with the top one?

Reply:
In fact... we don't 🖖